SolutionStack/sysdig/adoc/TRD-Kubernetes-ss-sysdig-gs.adoc

```diff
@@ -0,0 +1,247 @@
+
+
+To create your own, custom policy, first select Add Policy in the upper right corner.  Then, in the pop-up screen, select Workload Policy.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-choose-policy.png[scaledwidth="75%", align="center"]
+
+In the New Policy screen, enter a name and a description:
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-new-policy-name.png[scaledwidth="75%", align="center"]
+
+Keep the policy defaults: Enabled with severity level of High.i  You can customize the scope of this rule, indicating where it should apply.  In this example, the scope is contained to a namespace, called terminal-shell-in-container.  You can also set this to a broader scope, such as kubernetes.cluster.name, for example.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-policy-scope.png[scaledwidth="75%", align="center"]
+
+Next, you’ll want to assign a rule to use with the policy.  Select Import from Library.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-import.png[scaledwidth="75%", align="center"]
+
+Find and select "Terminal shell in a container" from the Rules Library, then click Import Rules.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-rules-library.png[scaledwidth="75%", align="center"]
+
+You should now see the "Terminal shell in a container" rule assigned to your policy.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-rule-config.png[scaledwidth="75%", align="center"]
+
+
+Next, under Actions, you can choose whether you’d like to kill, stop, or pause the container when the policy is triggered.  For this example, leave this set to Nothing.  Read more about Actions https://docs.sysdig.com/en/docs/sysdig-secure/policies/manage-policies/#define-actions[here].
+
+Your email is the default notification channel, but you can configure additional options under settings > notification channels. For example, you can choose to send notifications to a Slack Channel.  You can also leave this blank.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-actions-config.png[scaledwidth="75%", align="center"]
+
+Next, you can enable a Capture to record system call data surrounding an event. For this example, toggle on the capture, provide a file name, and choose the number of seconds before and after the event to record.  For example, 10 seconds before, and 20 seconds after the event.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-capture-config.png[scaledwidth="75%", align="center"]
+
+Finally, to create the policy, click Save.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-save-policy.png[scaledwidth="75%", align="center"]
+
+
+=== Trigger a Test Event
+
+You can trigger a test event to observe how Sysdig Secure detects the terminal shell in a container event and drill into the captured information to help you investigate the incident.
+
+First, open a terminal shell from any container that is running in your SUSE Rancher cluster.  Simply go to the Workloads section and get into any Pod with a running container.  Just for example purposes, the Sysdig agent pod will do.
```

Comment:
Shorten a little, make it a little less colloquial, consistently use lowercase for "pod" (I hope that is correct):

```suggestion
First, open a terminal shell from any container that is running in your SUSE Rancher cluster.  Go to the Workloads section and get into any pod with a running container. As an example, you can use the Sysdig agent pod.
```

I would also suggest to replace "get into" but dunno what with because I don't know the UI. E.g. "open the details page for any pod" or similar.

Reply:
Simpler is better ... thank you.  I didn't like "get into" either; I've changed it to "access."